src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -419,7 +419,7 @@ describe('StoryPromo Container', () => {
 
       it('should render a related item not contained within a list', () => {
         const { container } = render(
-          <WrappedStoryPromo item={[indexAlsosItem[0]]} topStory />,
+          <WrappedStoryPromo item={indexAlsosItem.relatedItems[0]} topStory />,
```

Comment:
add a line before like

`const onlyOneRelatedItem = { ...indexAlsosItem, relatedItems: [ indexAlsosItem.relatedItems[0] ] };`
(maybe way up near where all the other fixtures are defined)

then

```suggestion
          <WrappedStoryPromo item={onlyOneRelatedItem} topStory />,
```

cc @DenisHdz for double checking

Reply:
Ohhhhh `not contained within a list` means, the array has length 1 so doesnt include a `li` and `ul` elements, damn my FP knowledge is getting rusty haha